src/reanimated2/layoutReanimation/sharedTransitions/index.ts

```diff
@@ -53,7 +53,8 @@ export class SharedTransition implements ILayoutAnimationBuilder {
           const keyToTargetValue =
             'target' + propName.charAt(0).toUpperCase() + propName.slice(1);
           animations[propName] = withTiming(values[keyToTargetValue], {
-            duration: 1000,
+            // native screen transition takes around 500ms
+            duration: 500,
```

Comment:
I think it's 350 ms to be exact ([source](https://github.com/software-mansion/react-native-screens/blob/24393d3c1f9f2c5fdffd0f4ad3c8df9e221dc03f/ios/RNSScreenStackAnimator.mm#L8-L11)) but 500 ms is okay as well

Reply:
I set 500 ms because, in my subjective opinion, the transition with a 500 ms duration looks much better than animation with a 350 ms duration, but like I said. It is my subjective opinion. I think that we should allow for users to determine the default transition duration. We should do it in another PR.